en/firefox/features/fast-2023.ftl

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/fast/
+
+features-fast-is-firefox-the-fastest-browser = Is { -brand-name-firefox } the fastest browser?
+
+# HTML description
+features-fast-firefox-uses-less-memory-than = { -brand-name-firefox } uses less memory than Chrome, so your other programs can keep running at top speed.
+features-fast-browse-faster = Browse faster
+
+# The odd punctuation here is meant to represent how a computer can run slow.
+features-fast-every-computer-program-you-run = Every computer program you run takes up some memory. If your computer is low on memory, this can cause… a…… significant………… slow down. { -brand-name-firefox } aims for a balance – using enough memory to let you browse smoothly and leaving plenty of memory to keep your computer responsive.
+features-fast-multi-tasking-with-multiple-tabs = Multi-tasking with multiple tabs
+features-fast-firefox-is-a-multi-process-browser = { -brand-name-firefox } is a multi-process browser, which means that your tabs stay fresh and won’t take forever to reload. With 86% less lag time, you can switch quickly between tabs even when your little inner voice says you have waaay too many tabs open.
+features-fast-faster-gaming = Faster gaming
+
+# A "humblebrag" is something that appears to be modest or self-deprecating, but is actually a way of bragging. This is an un-humble brag because we're bragging openly. Ignore this string if there's no adequate translation.
```

Comment:
It wouldn't be shown at all. I thought this one was so specifically colloquial English that I suspect it may untranslatable in some cases, so I set it up as a separate string from the rest of the sentence and wrapped it in a conditional so it only shows if it's available. 
https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/features/fast.html#L23-L25

However, we could also hard-code it for en-US only and not even make it a string at all, if you think that's better.

Reply:
Thinking about it a bit more, the sentence reads just fine without this bit so maybe we should make it en-US-only after all. Asking localizers to leave a string untranslated feels weird and probably just makes it more confusing. I'll remove it.

Reply:
Indeed, we shouldn't ask localizers to keep an empty translation (it's confusing, it messes up translation memory).

Usually, we ask content to provide a less colloquial alternative in the comment (e.g. `Forgetting about false modesty:`).